shared/js/smart-screen/f_t_e_wizard.js

```diff
@@ -221,6 +221,16 @@
         // Do forced focus again on mouse click to prevent focus lost problem
         // when running on devices like PC and phone.
         this.focus();
+
+        switch (evt.target.dataset.behavior) {
```

Comment:
Looks ok, but since the switch-case here is duplicated with `keyup` event, I'd prefer to add a function like `perform(string)` and call it in both `click` and `keyup` event with argument `evt.target.dataset.behavior`.